components/signature/overview.md

```diff
@@ -0,0 +1,151 @@
+
+This article provides information about the <a href = "https://www.telerik.com/blazor-ui/signature" target="_blank">Blazor Signature component</a> and its core features.
+
+The Blazor Signature component provides an area where users can draw their signatures with the mouse, pen, or with their fingertips. You can use the component as part of any application where the users must sign a document. The Blazor Signature can be easily integrated with the [TelerikForm.]({%slug form-overview%}) The component stores the value of the signature as a <a href="https://learn.microsoft.com/en-us/dotnet/api/system.buffers.text.base64?view=net-6.0">base64 string</a>
+
+## Creating Blazor Signature
+
+1. Use the `TelerikSignature` tag to add the component to your razor page.
+1. Set the `Value` (one-way or two-way binding) parameter to a `string` property.
+1. Set the `Width` and `Height` parameters to `px` values.
+
+>caption Blazor Signature with basic configuration
+
+````CSHTML
+<TelerikSignature @bind-Value="@SignatureValue" Width="300px" Height="300px">
+</TelerikSignature>
+
+@code {
+    private string SignatureValue { get; set; }
+}
+````
+
+## Appearance
+
+The Signature component provides settings to control its appearance. [Read more about the Blazor Signature appearance settings...]({%slug signature-appearance%})
+
+## Events
+
+The Blazor Signature **blur** and value **change** events to respond to user actions. [Read more about the Blazor Signature events...]({%slug signature-events%})
+
+## Large Content Support
+
+In **Blazor server-side applications**, the Signature uses the **SignalR WebSocket**, which has a default maximum message size of **32 KB**. To work with larger content, increase the max WebSocket message size for the Blazor application:
+
+>caption Program.cs (.NET 6)
+
+<div class="skip-repl"></div>
+
+````CS
+using Microsoft.AspNetCore.SignalR;
+
+var builder = WebApplication.CreateBuilder(args);
+
+//...
+
+// SignalR message size for the Signature
+builder.Services.Configure<HubOptions>(options =>
+{
+    options.MaximumReceiveMessageSize = 1024 * 1024; // 1MB
+});
+````
+
+>caption Startup.cs (.NET 5)
+
+<div class="skip-repl"></div>
+
+````CS
+public void ConfigureServices(IServiceCollection services)
+{
+    //...
+
+    // SignalR message size for the Signature
+    services.Configure<HubOptions>(options =>
+    {
+        options.MaximumReceiveMessageSize = 1024 * 1024; // 1MB
+    });
+}
+````
+
+## Signature Parameters
+
+The following table lists the Signature parameters. Also check the [Signature API Reference](/blazor-ui/api/Telerik.Blazor.Components.Signature).
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `AriaDescribedBy` | `string` | Maps to the `area-describedby` attribute. Use this parameter to establish a relationship between widgets or groups and the text that describes them. |
```

Comment:
Start directly with what the parameter represents. By the way, what is "groups"?

```suggestion
| `AriaDescribedBy` | `string` | The `area-describedby` attribute. Use this parameter to establish a relationship between widgets or groups and the text that describes them. |
```

Reply:
Quite honestly, I have no idea, I did not understand what it does exactly, read that from the https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby#:~:text=The%20aria%2Ddescribedby%20attribute%20lists,not%20limited%20to%20form%20controls.

Reply:
In such cases, it may be better to just link the generic external docs and not deal with unclear content on our side.